lang/java/avro/src/main/java/org/apache/avro/file/DataFileReader.java

```diff
@@ -88,17 +88,17 @@
     return dreader;
   }
 
-  /** Construct a reader for a file. */
+  /** Construct a reader for a file. Please close resource files yourself. */
```

Comment:
Maybe it is a good idea to add an example here, using the `<pre>` tag: https://dzone.com/articles/a-guide-to-formatting-code-snippets-in-javadoc

```
try (FileReader<User> dataFileReader = DataFileReader.openReader(file, datumReader)) {
  // Consume the reader
} catch (IOException e) {
  throw new RuntimeIOException(e, "Failed to read metadata for file: %s", file);
}
```
WDYT?

Reply:
Of course, this is a good idea to be more instructive, I will modify it later.
Thanks for viewing.